pep/app/pods/components/tables/most-cited/component.ts

```diff
@@ -8,45 +13,112 @@ interface TablesMostCitedArgs {
 }
 
 export default class TablesMostCited extends Component<TablesMostCitedArgs> {
-    columns: ColumnValue[] = [
-        {
-            id: '0',
-            valuePath: 'title',
-            name: 'Title',
-            isFixedLeft: false,
-            width: 200,
-            staticWidth: 200,
-            maxWidth: 200,
-            minWidth: 200,
-            cellComponent: 'tables/cell/html',
-            isSortable: true
-        },
+    @service intl!: IntlService;
+    @service fastbootMedia!: FastbootMediaService;
+
+    /**
+     * Columns for the table. The `computed` is required
+     *
+     * @readonly
+     * @type {ColumnValue[]}
+     * @memberof TablesMostCited
+     */
+    @computed()
+    get columns(): ColumnValue[] {
+        return this.fastbootMedia.isSmallDevice
+            ? [
+                  {
+                      id: '0',
+                      valuePath: 'title',
+                      name: this.intl.t('mostCited.table.title'),
+                      isFixedLeft: false,
+                      width: 200,
+                      staticWidth: 200,
+                      cellComponent: 'tables/cell/most-viewed-publication',
+                      isSortable: true
```

Comment:
are we hitting the endpoint for server-side sorting when a column is sorted? If not i dont think we should enable client-side sorting since these tables are paginated right? Otherwise as the user scrolls down to load more records, all subsequent pages would either not be sorted, or there would be some weird re-arranging of rows as they scrolled down, to put them in the right order, I think.

Reply:
We are hitting the server when we filter